website/docs/whats-new/release-notes/partials/components.md

```diff
@@ -14,8 +14,6 @@
 
 ## 4.7.0
 
-[4.7.0 documentation](https://hds-website-4-7-0.vercel.app/)
-
```

Comment:
Blocker: Need to find out why these documentation links got deleted.

Reply:
I looked into this and it seems like we missed adding the docs links for 4.6.0 and 4.7.0 in the GitHub changelog (we only added them to the generated release notes for the website), so I had to rebase and add a commit to resolve this.

Reply:
@alex-ju thank you! ❤️